plugins/repository-hdfs/build.gradle

```diff
@@ -269,6 +290,10 @@ def getNonSecureNamenodePortForVersion(hadoopVersion) {
     return 10003 - (2 * hadoopVersion)
 }
 
+def getHttpPortForVersion(hadoopVersion) {
+  return 10004 - (2 * hadoopVersion)
```

Comment:
I tried running the tests locally and the fixture does not start due to an already used port. I think these port assignment functions need to be updated to use an offset of 3 instead of 2 now.